bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java

```diff
@@ -1671,10 +1671,13 @@ private void createHTML_58_NotImportedFiles(final StringBuilder sb, final ArrayL
          sb.append(deviceFile.size);
          sb.append(HTML_TD_END);
 
+         final EasyConfig easyConfig = getEasyConfig();
```

Comment:
When a "static" value is used in a loop, then I would move it out of the loop as this is computed for each iteration. In this case it could be neglected but I do it when possible.

Reply:
Oh, I didn't notice that, will fix.